python/ct/crypto/verify.py

```diff
@@ -50,6 +53,129 @@ def decode_signature(signature):
                                    len(remaining)))
     return (hash_algo, sig_algo, remaining)
 
+def get_issuer(chain):
```

Comment:
prefix with underscore to indicate private method and name _get_precertificate_issuer so it's clear this is useful in precertificate serialization context.